static/src/javascripts/projects/common/modules/experiments/tests/adblocking-response.js

```diff
@@ -73,13 +73,13 @@ define([
             id: 'control',
             test: function(){}
         }, {
-            id: '299',
+            id: '300',
             test: function () {
                 if (isQualified()) {
```

Comment:
You might want `isQualified` to be a part of the `canRun` check rather than the `test` function. As it is now, you'll track impressions for people that don't qualify, so your results for the variants with `isQualified` in test will be lower. 

If you move `isQualified` to `canRun`, you'll only fire impressions for people that do qualify, so your results won't have the negative bias that you'd see for the variants in their current form.